packages/vue/src/components/SpGetTxList/SpGetTxList.vue

```diff
@@ -81,9 +81,23 @@ export default defineComponent({
         .slice(0, state.listSize)
         .sort((a, b) => b.height - a.height)
     })
+
+    let filteredList = computed(() => {
+      return list.value.reduce((acc:TxForUI[], current) => {
+        const x = acc.find(item => {
+          return item.sender === current.sender && item.receiver === current.receiver && item.timestamp === current.timestamp
+        })
+        if (!x) {
```

Comment:
this should be moved to the normalizer function inside `useTxs` so to keep list length -- and by doing so `newTxs` as well -- concise

another point: let's use `hash` & `dir` as operands